### PR TITLE
Add capitalize func and tests for v.1.1.0

### PR DESCRIPTION
**Pull request by GoRunMaxim**

### CODE REVIEW

File: strings.go [hunk 1]
```diff
@@ -11,3 +13,14 @@ func Reverse(s string) string {
...
+// Capitalize func reverses and returns string
+func Capitalize(s string) interface{} {
+	if len(s) > 0 {
+		rns := []rune(s)
+		rns[0] = unicode.ToUpper(rns[0])
+		return string(rns)
+	}
+
+	return s
+}
```
Comment (papahmsolo) [inline strings.go:26]: Your comment says `Capitalize func reverses and returns string` but what does it actually return?

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Looks good!

Try some other tests, for instance, letters that don't have lower/uppercase or even non-letter synbols

File: strings.go (hunk 1 above)
Comment (GoRunMaxim) [reply strings.go:26]: **Letters that don't have lower/uppercase or even non-letter synbols**

I faced with such trouble: in Russian language letter 'ъ' doesn't have uppercase. But during unicode table it does. So what the solution should be?

**Review by papahmsolo — COMMENTED** [review]
lgtm!

**Review by papahmsolo — APPROVED** [review]

**Pull request merged**